crud/common/sharding/sharding_func.lua

```diff
@@ -43,13 +47,25 @@ end
 local function get_function_from_G(func_name)
     local chunks = string.split(func_name, '.')
     local sharding_func = _G
+    local sharding_module = false
+
+    if sharding_module_names[chunks[1]] then
+        sharding_func = require(chunks[1])
+        sharding_module = true
+        table.remove(chunks, 1)
+    end
 
     -- check is the each chunk an identifier
```

Comment:
I don't get how this "multiple chunk" load works (and not sure it is valid).

Reply:
Before this patch we got record by `chunk` key from `_G`, but this approach doesn't work with `vshard` because `require('vshard')` doesn't set `vshard` key in `_G`. So, in vshard case we got record directly from `vshard` table, not from `_G`.